pages/store/[nftClassId]/index.vue

```diff
@@ -298,11 +322,21 @@
             v-text="selectedPricingItem?.currency"
           />
           <span
+            v-if="isLikerPlus"
             class="text-2xl font-semibold"
-            v-text="selectedPricingItem?.formattedPrice"
+            v-text="selectedPricingItem?.discountedPrice"
+          />
+          <PlusBadgeIcon
```

Comment:
Not sure if `discountedPrice` would be a better condition for `v-if` here
i.e. `discountedPrice` might have other source than `plus` in the future